controls.py

```diff
@@ -1,11 +1,9 @@
 import dash
-from dash import dcc
 from dash import html
 import dash_bootstrap_components as dbc
-import dash_leaflet as dlf
 
 
-def Number(id, default, min, max):
+def Number(id, default, min=None, max=None):
```

Comment:
I'll take the blame for this, since I wrote this file original, but something I do wonder is if the order of the parameters and names of these components ought not to match the built-in dash components. My trainees found that very confusing that you set the id differently here.

Reply:
I am not sure I understand... Does dbc.Input cares about the order?

Reply:
What I'm saying is that the order of the parameters is inconsistent with how the built-in components work, which is a tad confusing in my opinion.

Reply:
I don't think I get it. I can see that you might want to use "value" for "default". But I don't think we want optional id and default so we would not be able to put value/default last. But I'm probably off topic.

Reply:
The `dbc.Input` constructor doesn't have any positional arguments, only keyword arguments. So (a) they're all optional, and (b) they can be provided in any order. In contrast, here we've made `id` and `default` positional arguments, which means they must come before everything else. I guess @kgraaf is asking whether we should make `id` and `default` optional?

Might be a good idea, but not worth holding up the PR.